.gitignore

```diff
@@ -4,3 +4,5 @@ lib_managed/
 src_managed/
 project/plugins/project/
 .idea
+.metals
+.vscode
```

Comment:
I'd add `.bsp` here (seems both IntelliJ and VsCode can generate this) and I've seen it sneak into various scaffolded repos